src/http/Operation.ts

```diff
@@ -24,6 +25,10 @@ export interface Operation {
    * Conditions the resource must fulfill for a valid operation.
    */
   conditions?: Conditions;
+  /**
+   * The credentials of the client requesting the current operation.
+   */
+  credentials?: Credentials;
```

Comment:
What is the core reason that operation needs to know about the credentials?
I thought that there was no dependency from the HTTP package on credentials; we might just have created one.

Reply:
If that reason is `Wac-Allow` then… well… 😅 We might need to find something else.
I see the performance shortcut though. Although caching might have our back there.